lib/tasks/import_bind.rake

```diff
@@ -25,11 +25,21 @@ task :import_bind do
     # Skip the SOA
     next if record.type == 'SOA'
 
+    if record.label == zone.origin or record.label == "@"
```

Comment:
Our [Ruby styleguide](https://github.com/alphagov/govuk-lint/blob/master/configs/rubocop/gds-ruby-styleguide.yml#L122-L125) says we prefer `||` to `or`.